lib/membrane/rtp/session_bin.ex

```diff
@@ -375,9 +384,31 @@ defmodule Membrane.RTP.SessionBin do
       }
     }
 
+    # workaround: as TWCC is a transport-wide extension, there should exist only one TWCC child
+    # that handles packets from all incoming streams that have declared support for it
+    {maybe_twcc, rtp_extensions} = Keyword.pop(rtp_extensions, :twcc)
+
+    should_link_twcc? = maybe_twcc != nil
+
+    {new_children, state} =
+      if should_link_twcc? and not Map.has_key?(ctx.children, :twcc) do
+        {twcc_ssrc, state} = add_ssrc(nil, state)
```

Comment:
Any specific reason for putting `nil` ssrc for the twcc in the state? Why do we need that mapping in the end?

Reply:
I needed a way to generate an SSRC for TWCC using mechanism that is already implemented (using SSRC generator passed as a `SessionBin` option and adding this SSRC to set of our local SSRCs). Unfortunately, this mechanism assumes that for every "local SSRC" (a one that is generated by `SessionBin`s generator) there is some "remote SSRC", what doesn't hold for TWCC (and probably for other transport-wide feedbacks too). As the first argument of `add_ssrc/2` is "remote SSRC", I pass `nil` to indicate that there is no remote SSRC associated with the local SSRC that will be generated.